workbench-core/example/infrastructure/src/example-stack.ts

```diff
@@ -708,9 +710,16 @@ export class ExampleStack extends Stack {
       domainPrefix: domainPrefix,
       websiteUrls: websiteUrls,
       userPoolName: userPoolName,
-      userPoolClientName: userPoolClientName,
+      webUiUserPoolClientName: `${userPoolClientName}-webUi`,
+      programmaticAccessUserPoolName: `${userPoolClientName}-iTest`,
       oidcIdentityProviders: [],
-      accessTokenValidity: Duration.minutes(60), // Extend access token expiration to 60 minutes to allow integration tests to run successfully. Once MAFoundation-310 has been implemented to allow multiple clientIds, we'll create a separate client for integration tests and the "main" client access token expiration time can be return to 15 minutes
+      // Extend access token expiration to 60 minutes to allow integration tests to run successfully. Once MAFoundation-310 has been implemented to allow multiple clientIds, we'll create a separate client for integration tests and the "main" client access token expiration time can be return to 15 minutes
+      webUiUserPoolTokenValidity: {
+        accessTokenValidity: Duration.minutes(60)
```

Comment:
Can we revert this back to the default value? Or does it have to wait until we have e2e tests to fully step away from using the webUi cleint for integ tests?

Reply:
I kept this value since I added one test that uses WebUI token (`routeHandler.test.refreshAccessToken`). My understanding is that we can remove this once we have E2E tests.